cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -529,6 +529,24 @@ rules:
       - "*"
     verbs:
       - "*"
+  - apiGroups:
+      - apps
+    resources:
+      - deployments/scale
+    verbs:
+      - patch
+  - apiGroups:
```

Comment:
It seems fine to combine this with the previous one so that `deployments/scale` and `deployments` can have both `patch` and `delete` access.